ci/libvcx-codecov.dockerfile

```diff
@@ -1,10 +1,6 @@
 FROM ubuntu:18.04 as BASE
 
 ARG UID=1000
-
-ARG INDYSDK_PATH=/home/indy/vdr-tools
```

Comment:
Please make these changes in other dockerfiles as well

Reply:
Removed this and all others mentions of var-tools.